### PR TITLE
Use parent or parents attributes of Pathlib Path object instead of chaining ".." strings

### LINKED ISSUE #532: Use `parent` or `parents` attributes of Pathlib Path object instead of chaining ".." strings
https://github.com/aramis-lab/clinicadl/blob/c6177e42a0ea7528b50a26864fe807d3f5e3cdb8/clinicadl/train/train_utils.py#L29

https://github.com/aramis-lab/clinicadl/blob/c6177e42a0ea7528b50a26864fe807d3f5e3cdb8/clinicadl/train/train_utils.py#L48

https://github.com/aramis-lab/clinicadl/blob/c6177e42a0ea7528b50a26864fe807d3f5e3cdb8/clinicadl/utils/maps_manager/maps_manager_utils.py#L22

### PR DESCRIPTION
**Pull request by thibaultdvx**
Fixes #532

### CODE REVIEW

File: clinicadl/hugging_face/hugging_face.py [hunk 1]
```diff
@@ -132,7 +132,7 @@ def create_readme(
     if not config_file.is_file():
         raise ClinicaDLArgumentError("There is no maps.json file in your repository.")
 
-    clinicadl_root_dir = (Path(__file__) / "../..").resolve()
+    clinicadl_root_dir = Path(__file__).parents[1]
```
Comment (NicolasGensollen) [inline clinicadl/hugging_face/hugging_face.py:135]: Even if not specified in the linked issue, I think we can also simplify the line just after this one.
Indeed, `clinical_root_dir` is already a Path object, such that there is no need to cast it.

File: clinicadl/train/train_utils.py [hunk 2]
```diff
@@ -26,7 +26,7 @@ def build_train_dict(config_file: Path, task: str) -> Dict[str, Any]:
     """
     if config_file is None:
         # read default values
-        clinicadl_root_dir = (Path(__file__) / "../..").resolve()
+        clinicadl_root_dir = Path(__file__).parents[1]
```
Comment (NicolasGensollen) [inline clinicadl/train/train_utils.py:29]: Same here

File: clinicadl/train/train_utils.py [hunk 3]
```diff
@@ -45,7 +45,7 @@ def build_train_dict(config_file: Path, task: str) -> Dict[str, Any]:
             del user_dict["Random_Search"]
 
         # read default values
-        clinicadl_root_dir = (Path(__file__) / "../..").resolve()
+        clinicadl_root_dir = Path(__file__).parents[1]
```
Comment (NicolasGensollen) [inline clinicadl/train/train_utils.py:46]: And here

**Review by NicolasGensollen — COMMENTED** [review]
LGTM, thanks @thibaultdvx !
I think we can also make another small improvement while we are at it (see comments).

**Comment by thibaultdvx** [comment]
Done @NicolasGensollen

**Review by NicolasGensollen — APPROVED** [review]
LGTM, thanks !

**Pull request merged**